src/main/kotlin/blackjack/domain/BettingMoney.kt

```diff
@@ -0,0 +1,15 @@
+package blackjack.domain
+
+data class BettingMoney(val money: Int) {
+    constructor(money: String) : this(money.toInt()) {
+        validateNumber(money)
+    }
+
+    constructor(money: Double) : this(money.toInt())
+
+    fun multiply(rate: Double) = BettingMoney(money * rate)
```

Comment:
operator overloading을 사용해보는건 어떨까요?

```suggestion
    operator fun times(rate: Double) = BettingMoney(money * rate)
```

https://kotlinlang.org/docs/operator-overloading.html#arithmetic-operators

Reply:
배워놓고 안쓰고있었네요😅
덕분에 적용했습니다 감사합니다~